playbook/app/pb_kits/playbook/pb_timestamp/_timestamp.jsx

```diff
@@ -82,16 +82,14 @@ const Timestamp = (props: TimestampProps) => {
     return `Last updated ${userDisplay} ${dateTimestamp.value.fromNow()}`
   }
 
-  const datetimeOrText = timestamp ? fullDateDisplay() : text
```

Comment:
Setting this variable here isn't good as it triggered a second timezone string to be added when it wasn't needed. The variable `datetimeOrText` had only one usage within a `case` statement which means it is only needed there. Since our linter doesn't like lexical variables in case statements, I deferred to making one ternary inline.